vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/Microsoft.ReactNative.Managed.CodeGen.UnitTests.csproj

```diff
@@ -49,8 +49,14 @@
 
   <Target Name="EnsureRuntimeDependenciesAreBuilt" AfterTargets="Build">
```

Comment:
>EnsureRuntimeDependenciesAreBuilt [](start = 16, length = 33)

Why not to have normal project references instead?

Reply:
Because these projects are NetCore3.1 and our other projects are UAP, The Msbuild SDK's are not happy about that...

Reply:
And they are not really a code dependency, just an order-dependency to ensure that the runtime of the unittest has the files available so it can read those files.